alchemy-web/docs/guides/cloudflare-nuxt-pipeline.md

```diff
@@ -119,10 +119,10 @@ await app.finalize();
 
 ## Infer Binding Types
 
-Create an `src/env.d.ts` file to support type hints for Cloudflare bindings:
+Create an `env.d.ts` file to support type hints for Cloudflare bindings:
 
 ```typescript
-// src/env.d.ts
+// env.d.ts
 /// <reference types="@cloudflare/workers-types" />
```

Comment:
Oops, I think this reference is wrong. 

Should be env.d.ts instead of worker-types. 

See: https://github.com/sam-goodwin/alchemy/blob/main/examples/cloudflare-vite/src/env.d.ts

Reply:
It might also be worth checking if we can add this to types in tsconfig and have it apply globally.